_pages/en/it-picture-medium-term.md

```diff
@@ -123,7 +135,9 @@ The security team is involved in the design phase for all projects and a securit
 Security tests are built into the automated testing process and can identify common security vulnerabilities.
 Theses security tests can be applied uniformly as a part of a continuous integration pipeline or build process.
 
-En conclusion, ...
+The device provisioning division (e.g. laptops) is experimenting with new methods of device provisioning, including desktop as a service, virtual desktops in the cloud, and automatic patching from multiple trusted sources (e.g. directly from MS Azure for Microsoft Windows OS).
+
+ESDC is experimenting with different solutions to solve its network bandwidth solutions. This includes evaluating alternate suppliers than SSC, promoting teleworking, and re-architecting its solutions to reduce latency experience for end-users (TODO: link to network modernization initiative).
```

Comment:
Too specific for the current version, I think

But would be good to think how we could include the things IT (Applications and Infrastructure (Devices, servers, and network)

Ex.: Teams are able to provision device (laptops, phones, ..) or environment (servers, software in an automated fashion, and know that any new device or environment reproduced from the same configuration is identical.

Reply:
I wanted to present a view of the consumer of IT (i.e. the other 26k employees). It was one thing AG wanted to be represented. For a medium term view (something we can see happening in the near future) we should describe where this is going. 

I recommend keeping it and engaging both Enterprise Ops (where desktop computing resides) and SSC's WTD WG on where they're seeing this going (e.g. windows patching directly from Azure): https://gcconnex.gc.ca/groups/profile/12903340/wtd-common-desktop-operating-environment-environnement-dexploitation-commun-des-ordinateurs-de-bureau-des-atmt